source/jobs/data-verification/use-data-verification.txt

```diff
@@ -74,11 +74,11 @@ A data verification process can be any of the following statuses:
    * - ABORTED
```

Comment:
(consistent terminology)
While you're in here, you might consider switching from "validation" to "verification" throughout the page.

Reply:
Done.